cmd/minishift/cmd/start.go

```diff
@@ -315,58 +329,6 @@ func automountHostfolders(driver drivers.Driver) {
 	}
 }
 
-// Set Docker Proxy
-func setDockerProxy() {
-	if viper.IsSet(httpProxy) {
-		dockerEnv = append(dockerEnv, fmt.Sprintf("HTTP_PROXY=%s", viper.GetString(httpProxy)))
-	}
-	if viper.IsSet(httpsProxy) {
-		dockerEnv = append(dockerEnv, fmt.Sprintf("HTTPS_PROXY=%s", viper.GetString(httpsProxy)))
-	}
-	if viper.IsSet(noProxyList) {
-		dockerEnv = append(dockerEnv, fmt.Sprintf("NO_PROXY=%s,%s", updateNoProxyForDocker(),
-			viper.GetString(noProxyList)))
-	} else if viper.IsSet(httpProxy) || viper.IsSet(httpsProxy) {
```

Comment:
@hferentschik you are right `--no-proxy` also should be used for `NO_PROXY` if proxy is specified. I might missed it when implementing this feature :( Good catch.